fcrepo-serialization/src/main/java/org/fcrepo/camel/serialization/SerializationRouter.java

```diff
@@ -129,4 +135,17 @@ public void configure() throws Exception {
                 "{{serialization.binaries}}${headers[CamelSerializationPath]}")
             .to("exec:rm");
     }
+
+    private List<Predicate> getUriFilter() {
```

Comment:
@acoburn Maybe `ProcessorUtils` in `fcrepo-camel`?   Would it be useful for people building their own routes that need to filter stuff similarly?   If we don't want people using it approach, then it's probably fine to keep as-is.  If  moving it to fcrepo-camel does makes sense, that could probably be done later too under a separate issue